src/Akeneo/Category/front/src/feature/components/templates/AddTemplateAttributeModal.tsx

```diff
@@ -0,0 +1,96 @@
+import {translate, useTranslate} from '@akeneo-pim-community/shared';
+import {useState} from "react";
+import {useCreateAttribute} from "../../hooks/useCreateAttribute";
+import {userContext} from "@akeneo-pim-community/shared/lib/dependencies/user-context";
+import {TextInput} from "akeneo-design-system/lib/components/Input/TextInput/TextInput";
+import {AttributesIllustration, Button, Field, Helper, Link, Modal} from "akeneo-design-system";
```

Comment:
Your IDE auto-import configuration is not doing a good work :thinking:
It shouldn't import files from deep into the library folders.
```suggestion
import {AttributesIllustration, Button, Field, Helper, Link, Modal, TextInput} from "akeneo-design-system";
```